test/safe/workflow.test.ts

```diff
@@ -12,70 +34,54 @@ describe("GitHub Actions", () => {
 		it("should update the Jira issue with the linked GitHub workflow_run", async () => {
 			const fixture = require("../fixtures/workflow-basic.json");
 
-			githubNock
-				.get("/users/test-workflow-run-user-login")
-				.reply(200, {
-					login: "test-workflow-run-author-login",
-					avatar_url: "test-workflow-run-author-avatar",
-					html_url: "test-workflow-run-author-url",
-				})
-				.get("/repos/test-repo-owner/test-repo-name/check-runs/1")
-				.reply(200, [
-					{
-						id: 1,
-						html_url: "test-repo-url",
-						status: "queued",
-						conclusion: null,
-						app: {
-							id: 1234,
-							slug: "github-actions",
-						},
-					},
-				]);
-
-			jiraNock.get("/rest/api/latest/issue/TEST-123?fields=summary")
+			// TODO: need to start validating all issue keys with jira
+			/*jiraNock.get("/rest/api/latest/issue/TEST-123?fields=summary")
 				.reply(200, {
```

Comment:
I wonder why it is commented? Don't we call this endpoint when we process webhooks?

Reply:
Yeah, it's a problem I found.  We're not validating issuekeys with jira in some instances.  Rachelle is doing a PR to fix that.